readme.md

```diff
@@ -71,7 +71,17 @@ Kindly ask you to raise the issue in case of any suggestions/bugs.
    ```bash
     java -jar docker-unittests.jar -f image-tests.yml -i openjdk:9.0.1-11
    ``` 
-   ![docker image tests results](./docs/image-tests-results.png)
+    ![docker image tests results](./docs/image-tests-results.png)
+
+#### General image test with output to xml file
+1. Add -o xml option to generate xml file
+   ```bash
+    java -jar docker-unittests.jar -f image-tests.yml -i openjdk:9.0.1-11 -o xml
+   ``` 
+    ![xml result of docker image testing](./docs/test-result-in-xml-1.png)
```

Comment:
Make the snapshots from IDEA or sublime text editor without `-` symbols.

Reply:
@bodax, Use https://www.sublimetext.com editor for snapshots 
![image](https://user-images.githubusercontent.com/1651114/50570276-20f46600-0daa-11e9-8b1c-50be41a12b09.png)